### PR TITLE
ISE-183 OpenIMIS Sonar - Backend Module

### PR DESCRIPTION
**Pull request by kpavlenkoSoldevelo**
fix for .flake8 configuration

### CODE REVIEW

**Review by dborowiecki — CHANGES_REQUESTED** [review]
The CI is failing. It seems like nose test executor is treating test_helpers as tests and tries to execute them. Please alternate django-nose configuration in order to skip helpers.

File: openIMIS/openIMIS/settings.py [hunk 1]
```diff
@@ -158,7 +159,7 @@ def SITE_URL():
     "django_apscheduler",
     "channels",  # Websocket support
     "developer_tools",
-    "drf_spectacular"  # Swagger UI for FHIR API
+    "drf_spectacular",  # Swagger UI for FHIR API
```
Comment (pgesek) [inline openIMIS/openIMIS/settings.py:162]: Remove this

Comment (pgesek) [reply openIMIS/openIMIS/settings.py:162]: The comma I mean

**Review by dborowiecki — APPROVED** [review]
LGTM

**Review by dborowiecki — APPROVED** [review]

**Pull request merged**